src/quickpicks/schemas.ts

```diff
@@ -106,12 +106,11 @@ export async function schemaVersionQuickPick(
   subject: string,
 ): Promise<Schema | undefined> {
   const loader = ResourceLoader.getInstance(schemaRegistry.connectionId);
-  const schemaVersions: Schema[] = await loader.getSchemasForEnvironmentId(
+  const schemasMatchingSubject: Schema[] = await loader.getSchemaSubjectGroup(
```

Comment:
Minor: could be helpful to rename that method to `getSchemasForSubject` or `getSubjectSchemas` -- re-reading this I thought it was going to return a `Subject`